keyboards/custommk/ergosentry_ansi/ergosentry_ansi.c

```diff
@@ -0,0 +1,51 @@
+// Copyright 2025 customMK
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include <stdbool.h>
+#include "keyboards/custommk/cmk_cm6646.h"
+
+typedef union {
+    uint32_t raw;
+    struct {
+        bool usb_audio_enabled;
+    };
+} user_config_t;
+
+user_config_t user_config;
+
```

Comment:
`keyboard_config_t keyboard_config;` or the like would be more correct, as this isn't really a) at the user level, b) user config but keyboard config.